Gems/ROS2/Code/Source/Odometry/ROS2OdometryCovariance.cpp

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "ROS2OdometryCovariance.h"
+#include "AzCore/RTTI/RTTIMacros.h"
+#include "AzCore/base.h"
+#include <AzCore/Serialization/EditContext.h>
+#include <AzCore/Serialization/EditContextConstants.inl>
+#include <array>
```

Comment:
nit: Should use AZStd::array instead
```suggestion
#include <AzCore/std/array.h>
```